config/main.py

```diff
@@ -5852,6 +5852,47 @@ def disable_link_local(ctx):
 
 
 #
+# 'static-anycast-gateway' group ('config static-anycast-gateway ...')
+#
+@config.group(cls=clicommon.AbbreviationGroup, name='static-anycast-gateway')
+def static_anycast_gateway():
+    """sag-related configuration tasks"""
+    pass
+
+#
+# 'static-anycast-gateway mac_address' group
+#
+@static_anycast_gateway.group(cls=clicommon.AbbreviationGroup, name='mac_address')
+def mac_address():
+    """Add/Delete static-anycast-gateway mac address"""
+    pass
+
+@mac_address.command('add')
+@click.argument('mac_address', metavar='<mac_address>', required=True, type=str)
+@clicommon.pass_db
+def add_mac(db, mac_address):
+    """Add static-anycast-gateway mac address command"""
+    log.log_info(f"'static-anycast-gateway mac_address add {mac_address}' executing...")
+
+    if not db.cfgdb.get_entry('SAG', 'GLOBAL'):
```

Comment:
nit: Is it possible to add a check to validate the MAC address (to be only unicast)

Reply:
Added MAC address format checking and also only allow unicast MAC address.